upgrade/util/util.go

```diff
@@ -1009,7 +1027,7 @@ func updateBackups(namespace string, lhClient *lhclientset.Clientset, backups ma
 	return nil
 }
 
-func updateBackupBackingImages(namespace string, lhClient *lhclientset.Clientset, backupBackingImages map[string]*longhorn.BackupBackingImage) error {
+func updateBackupBackingImages(namespace string, lhClient *lhclientset.Clientset, backupBackingImages map[string]*longhorn.BackupBackingImage, forceWrite bool) error {
```

Comment:
Actually we don't need to force update the following resources during 1.8.x to 1.9.0 upgrade, but still add `forceWrite` here. It should have not side effect during runtime because we don't update the content of these CRs:

- BackupBackingImage
- Orphan
- Snapshot
- SupportBundle
- SystemBackup
- SystemRestore
- VolumeAttachment